src/components/PromptForm/PromptForm.js

```diff
@@ -23,7 +23,7 @@ class PromptForm extends Component {
         return (
             <div className="prompt-form-section">
                 <form>
-                <textarea className="writing-prompt" rows="10" cols="100">
+                <textarea className="writing-prompt" placeholder='Keep writing...' rows="10" cols="100" name='prompt'>
```

Comment:
Since we may not get to the extension where a user can add their writing, we should remove this text box for now, or move it to another page for when we're ready to use it